kevm-pyk/src/kevm_pyk/foundry.py

```diff
@@ -675,7 +675,7 @@ def foundry_list(foundry_root: Path) -> list[str]:
         proof_digest = foundry.proof_digest(contract_name, test_name)
         if APRProof.proof_exists(proof_digest, apr_proofs_dir):
             apr_proof = APRProof.read_proof(proof_digest, apr_proofs_dir)
-            lines.extend(apr_proof.summary)
+            lines.extend(str(apr_proof.summary))
```

Comment:
@anvacaru is this the correct fix? Should we be calling instead `apr_proof.summary.lines`? https://github.com/runtimeverification/pyk/blob/dd38446e03ba1863bacb354a8a91b6b1146e49ec/src/pyk/proof/reachability.py#L629